imports/ui/pages/profile/profile.html

```diff
@@ -1,63 +1,51 @@
 <template name="profile">
   <div class="internals profile">
-    {{#if currentUser }} {{#isAuthorized}}
```

Comment:
Just want to make sure: after these changes, is there still no scroll on the profile when embedded in twitter?

Reply:
Yes, @bent0b0x. Did you get something?

Reply:
Nope, just confirming!

Reply:
Twitter embed is our reference now 😃